### DISCUSSION TITLE
Apple ID revoke token

### DISCUSSION THREAD

**Discussion by phoheisel** [capawesome-team/capacitor-firebase#279] [Q&A] [answered]
Apple rejected my iOS App because of a missing Apple ID revoke when users delete their account in my app.

Firebase itself seems to work on a solution here: https://github.com/firebase/firebase-ios-sdk/issues/9906
Can anyone point out, how I can implement the interims solution found here: https://github.com/jooyoungho/apple-token-revoke-in-firebase 
My problem there is where to get the authorizationCode to send to my "revokeToken" function as described in the solution above.

Thanks for any help!

**Comment by robingenz** [comment, marked as answer]
The `authorizationCode` is returned on sign-in:
```ts
const signInWithApple = async () => {
  const result = await FirebaseAuthentication.signInWithApple();
  console.log(result.credential?.authorizationCode);
};
```

**Reply by phoheisel** [reply]
thank you for the fast reply that looks very promising!
Should have been able to find that but was looking at the wrong place..
